lib/v2/ymgal/game.js

```diff
@@ -0,0 +1,46 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const host = 'https://www.ymgal.games';
+
+const date = new Date();
+const year = date.getFullYear();
+const month = date.getMonth() + 1;
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: `${host}/release-list/${year}/${month}`,
+    });
+
+    const $ = cheerio.load(response.data);
+    const list = $('.game-view-card').toArray();
+
+    const items =
+        list &&
+        list.map((item) => {
+            item = $(item);
+            const itemPicUrl = item.find('.lazy').first().attr('data-original');
+            let info = '<ul>';
+            let tags = item.find('.ant-tag');
+            let tagslength = tags.length;
+            while (tagslength) {
+                info += `<li>${tags.first().text()}</li>`;
+                tags = tags.next();
+                tagslength--;
+            }
+            info += '</ul>';
```

Comment:
Please use `art-template` for rendering custom HTML content.
docs: https://docs.rsshub.app/joinus/script-standard.html#template